steam-buddy@.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
do we still need the other service file?

Reply:
Good question. Possibly not, but if you still want to be able to run as root you may want to keep it. I haven't really tested running the new service file as root and running it as steam-buddy@root is a bit weird. It probably does work, though.